tools/wake-format/emitter.cpp

```diff
@@ -619,7 +651,40 @@ wcl::doc Emitter::walk_package(ctx_t ctx, CSTElement node) {
 
 wcl::doc Emitter::walk_paren(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
-  MEMO_RET(walk_placeholder(ctx, node));
+  assert(node.id() == CST_PAREN);
+
+  auto comment_fmt =
```

Comment:
Handling comments makes me want to cry lol

Reply:
yeeeeeep, it is very hard to get right and it is going to make everythign 5x harder.

Easy solution, we should just remove comments from wake.

Reply:
lol